.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -0,0 +1,246 @@
+      with:
+        fetch-depth: 0
+        ref: ${{ env.ROX_GIT_REF }}
+
+    - uses: ./.github/actions/job-preamble
+      with:
+        gcp-account: ${{ secrets.GCP_SERVICE_ACCOUNT_STACKROX_CI }}
+
+    - name: Build updater
+      run: |
+        echo "Building updater for version ${{ env.SCANNER_BUNDLE_VERSION }} on branch ${{ env.ROX_GIT_REF }}..."
+        make tag
+        make -C scanner bin/updater
+
+    - name: Download NVD
+      uses: ./.github/actions/download-artifact-with-retry
+      with:
+        name: nvd
+        path: nvd.zip
+
+    - name: Set Updater
+      run: |
+        chmod +x scanner/bin/updater
+        mkdir vulns
+
+    - name: Create Folder
+      run: mkdir -p definitions/${{ env.SCANNER_BUNDLE_VERSION }}
+
+    - name: Run Updater (single bundle)
+      env:
+        STACKROX_NVD_ZIP_PATH: nvd.zip
+      run: |
+        scanner/bin/updater export vulns --manual-url "${{ needs.prepare-environment.outputs.manual_url }}"
+        mv vulns/* definitions/${{ env.SCANNER_BUNDLE_VERSION }}/
+
+    - name: Run updater (multi bundle)
+      env:
+        STACKROX_NVD_ZIP_PATH: nvd.zip
+      run: |
+        scanner/bin/updater export --manual-url "${{ needs.prepare-environment.outputs.manual_url }}" --split bundles
+        zip definitions/${{ env.SCANNER_BUNDLE_VERSION }}/vulnerabilities.zip bundles/*.json.zst
+
+    - name: Upload definitions artifacts
+      uses: actions/upload-artifact@v4
+      with:
+        name: artifact_${{ env.SCANNER_BUNDLE_VERSION }}
+        path: definitions
+
+  upload-definitions:
+    needs:
+    - build-and-run
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Download definitions artifacts
+      uses: actions/download-artifact@v4
+      with:
+        path: downloaded_artifacts
+
+    - name: Move files artifacts
+      run: rsync -av downloaded_artifacts/*/ definitions_files/
+
+    - name: Authenticate with Google Cloud
+      if: github.event_name == 'schedule'
+      uses: google-github-actions/auth@v2
+      with:
+        credentials_json: ${{ secrets.GOOGLE_SA_STACKROX_HUB_VULN_DUMP_UPLOADER }}
+
+    - name: Authenticate with Google Cloud
+      if: github.event_name == 'pull_request'
+      uses: google-github-actions/auth@v2
+      with:
+        credentials_json: ${{ secrets.GOOGLE_SA_CIRCLECI_SCANNER }}
+
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v2
+
+    - name: Upload to GCS
+      if: github.event_name == 'schedule'
+      run: |
+        ls -lr definitions_files
+        for dir in definitions_files/*; do
+          if [ -d "$dir" ]; then
+            gsutil -m cp -r "$dir" gs://definitions.stackrox.io/v4/vulnerability-bundles/
+          fi
+        done
+        gsutil cp -r gs://definitions.stackrox.io/v4/vulnerability-bundles/dev/* gs://definitions.stackrox.io/v4/vulnerability-bundles/1.0.0/
+
+    - name: Update PR vulnerabilities
+      if: github.event_name == 'pull_request'
```

Comment:
Notice that a workflow dispatch call won't match this, not sure if it was intentional?

Reply:
I added workflow dispatch call